### PR TITLE
Yssa about update

### PR DESCRIPTION
**Pull request by YPangilinan**
1. Created a new member description component to use on the members page
2. Added mock data for all changeable items to be replaced later with data from AWS
3. Added route to nav items
4. Added to header links

NOTES FOR SUGGESTIONS: 
- I guessed on the naming, so those can be changed as well as the route names and images
- For the headshots, I am not 100% sure that they will be square when we load in the other images. Any suggestions would be awesome!
- Had some issues adding the team page to the gatsby-node create pages so left it out for now. Probably need some assistance there!

### CODE REVIEW

File: src/templates/teampage.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import { GatsbyImage } from 'gatsby-plugin-image';
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+import MemberDescription from '../components/MemberDescription';
+import { memberData } from '../data/mockMemberData';
+
+const TeamPage = ({ pageContext: { teamPageData } }) => (
+    //TODO[CHRIS FROM YSSA]: ^^ about page data will need to be changed to utilize member data
+    <>
+        <Header />
```
Comment (YPangilinan) [inline src/templates/teampage.js:12]: Had to import Header and Footer individually versus using the Layout Component to fit the look of the mocks

File: package.json [hunk 2]
```diff
@@ -48,7 +49,9 @@
         "react-dom": "^17.0.1",
         "react-helmet": "^6.1.0",
         "sass": "^1.49.9",
+        "sharp": "^0.29.3",
```
Comment (cjdeleon62) [inline package.json]: just wondering if there was a reason sharp, babel-eslint, and typescript are being added?

File: src/components/MemberDescription.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    speaking,
+}) => {
+    return (
+        <div className="overflow-hidden bg-white">
+            <div className="relative mx-auto max-w-6xl py-14 px-4 sm:px-6 lg:px-8">
+                <div className="mt-7 lg:grid gap-0 lg:grid-cols-2">
+                    <div className="relative ml-8 mt-4 lg:col-start-1 lg:row-start-1">
+                        <div className="relative mx-auto max-w-prose text-base lg:max-w-none">
+                            <figure>
+                                <img
```
Comment (cjdeleon62) [inline src/components/MemberDescription.js:20]: yeah I think once we get an idea of how images will be handed to us via the API we are going to build, then we can worry about the sizing but this is good for now 👍

File: src/templates/teampage.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { GatsbyImage } from 'gatsby-plugin-image';
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+import MemberDescription from '../components/MemberDescription';
+import { memberData } from '../data/mockMemberData';
+
+const TeamPage = ({ pageContext: { teamPageData } }) => (
+    //TODO[CHRIS FROM YSSA]: ^^ about page data will need to be changed to utilize member data
```
Comment (cjdeleon62) [inline src/templates/teampage.js:10]: gotcha good todos commented throughout

File: src/templates/teampage.js (hunk 1 above)
Comment (cjdeleon62) [reply src/templates/teampage.js:12]: ah okay, we should probably get the designs for each page layout to follow some sort of format so we don't have to import them separately and have the padding not be unified between pages. We can def leave it for now and investigate later.

File: src/components/MemberDescription.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                                <h4 className="mt-2 font-bold text-faster-green">
+                                    COMPANIES
+                                </h4>
+                                {companies?.map((company) => (
+                                    <h5 key={company}>{company}</h5>
+                                ))}
+                            </div>
+                        </div>
+                    </div>
+                    <div className="mr-8 -ml-24 lg:mt-0">
```
Comment (cjdeleon62) [inline src/components/MemberDescription.js]: ah I think for this we may have to add some extra breakpoint at smaller viewports. It's causing text to go too far left at smaller breakpoints

**Review by cjdeleon62 — CHANGES_REQUESTED** [review]
Amazing work on this! 🎉 just saw some minor things but overall this is really good

File: package.json (hunk 2 above)
Comment (YPangilinan) [reply package.json]: For this one: these were things that my computer was missing so it got added to the package.json in order for me to run it. I can remove it if you want!

File: src/components/MemberDescription.js (hunk 5 above)
Comment (YPangilinan) [reply src/components/MemberDescription.js]: Fixed!

**Review by cjdeleon62 — APPROVED** [review]
LGTM! 🚀

**Pull request merged**